GREEDY/Extensions/GetReceiptExtensions.cs

```diff
@@ -13,7 +13,18 @@ public static List<string> GetLinesOfText(this Emgu.CV.OCR.Tesseract source)
 
         public static List<string> GetLinesOfText(this Page source)
         {
-            return source.GetText().Split('\n').ToList();
+            var PercentageMatched = source.GetMeanConfidence();
+            if (PercentageMatched > 0.5)
```

Comment:
If you are not putting this value in config (which I would totally recommend) than atleast write a comment what it means. Values like these make the code unmaintainable (imagine that I only now joined this project and have no idea what is going on and need to change stuff like this)